exporter/awsemfexporter/metric_translator.go

```diff
@@ -28,9 +28,8 @@ import (
 
 const (
 	// OTel instrumentation lib name as dimension
-	oTellibDimensionKey          = "OTelLib"
-	defaultNamespace             = "default"
-	noInstrumentationLibraryName = "Undefined"
+	oTellibDimensionKey = "OTelLib"
+	defaultNamespace    = "default"
```

Comment:
It's not clear to me why the changes in this file are made.  How do they relate to exporting quantiles?

Reply:
I delete the `noInstrumentationLibraryName= Undefined ` since [it is being used to know](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/17265/files#diff-7ab9fa22126292a4cb18c74ec6309729f3fc05049b7e5889dd80266db71f8f43R222-R223) if there are any instrumental library and the `fmt` doing the rest. However, we can check if the value is empty instead of passing the value `Undefinied`